core/identity-hub-verifier/src/main/java/org/eclipse/edc/identityhub/verifier/CredentialsVerifierExtension.java

```diff
@@ -48,8 +54,14 @@ public String name() {
 
     @Provider
     public CredentialsVerifier createCredentialsVerifier() {
-        var client = new IdentityHubClientImpl(httpClient, typeManager.getMapper(), monitor);
-        var verifiableCredentialsJwtService = new VerifiableCredentialsJwtServiceImpl(typeManager.getMapper(), monitor);
-        return new IdentityHubCredentialsVerifier(client, monitor, jwtCredentialsVerifier, verifiableCredentialsJwtService);
+        var client = new IdentityHubClientImpl(httpClient, typeManager.getMapper(), monitor, credentialEnvelopeTransformerRegistry);
+        return new IdentityHubCredentialsVerifier(client, monitor, credentialVerifierRegistry);
     }
+
+
+    @Provider(isDefault = true)
```

Comment:
Do we really this to be a default? 
For me there is no way somebody uses another implementation of `CredentialVerifierRegistry`